gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/rules/AccumulativeExecutionEngineRule.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
the  execution engine decision is taken after looking into the queryInfo  object recursively and the decision cant be taken based on a single queryInfo object in the tree so it is accumulative decision. If you have any better name for it,  will rename it later.